CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # API Mediation Layer Changelog
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
+## `1.19.0`
+
+- Feature: Configure the connection limit for the GW [#843](https://github.com/zowe/api-layer/issues/843)
+- Feature: Minimize size of API Mediation Layer [#909](https://github.com/zowe/api-layer/issues/909)
+- Feature: Allow removal or configuration of a service as a catalog [#727](https://github.com/zowe/api-layer/issues/727)
+- Bugfix: API ID is not sent to Eureka in metadata by Java enabler [#991](https://github.com/zowe/api-layer/issues/991)
+- Feature: Improve connection limits [#987](https://github.com/zowe/api-layer/issues/987)
+- Feature: SAF authorization checking code [#993](https://github.com/zowe/api-layer/issues/993)
+- Feature: Enhance connection limit log messages [#987](https://github.com/zowe/api-layer/issues/987)
+- Bugfix: Fix stuck opened tcp connections [#1009](https://github.com/zowe/api-layer/issues/1009)
+- Feature: Add /api/v1/gateway/services/{serviceId} endpoint that provides information about a service  in API ML for API clients (Disabled by default) [#873](https://github.com/zowe/api-layer/issues/873)
+- Feature: Support the Size limitation in the InMemory cache [#998](https://github.com/zowe/api-layer/issues/998)
```

Comment:
What is the benefit of supporting the size limitation in the InMemory cache?
Is a message is presented to the client when inserting record that has more data than allowed?
Has Data eviction been implemented so that the cache does not get filled with data?

Reply:
The data eviction was implemented and therefore the benefit is that when you reach the limit, it will handle the case properly.